cypress/tests/scroll-to-contest.js

```diff
@@ -0,0 +1,35 @@
+describe('Scroll Buttons', () => {
+  it('Scroll buttons do not appear on smaller screens', () => {
+    cy.visit('/#sample')
+    cy.getByTestId('activation-code').type('MyVoiceIsMyPassword', {
+      force: true,
+    })
+    cy.contains('Submit').click({ force: true })
+    cy.contains('Get Started').click()
+    cy.contains('Next').click()
+    cy.contains('Next').click()
+    cy.contains('Next').click()
+    cy.contains('Next').click()
+    cy.contains('Next').click()
+    cy.contains('Next').click()
+    cy.contains('Next').click()
+    cy.contains('Next').click()
```

Comment:
any reason these aren't just a `for` loop with a count?

Reply:
I had thought it may not be possible given the async nature of Cypress testing, and also wanted to get solution ASAP late last night. Just tried a loop and confirmed it works… so I'll merge it in.

Reply:
https://github.com/votingworks/bmd/commit/39b398402640c70198504239ba6a04d5a0704db5#diff-a9411614a0221919320a51141224572cR13